tests/CMakeLists.txt

```
@@ -75,6 +75,7 @@ function(add_unity8_unittest COMPONENT_NAME TARGET)
         ENVIRONMENT ${environment}
                     QT_QPA_PLATFORM=minimal
                     ${U8TEST_ENVIRONMENT}
+                    UITK_ICON_THEME=suru
```

Pedantic, but it might look nicer to have hard-coded variables together.

```suggestion
                    QT_QPA_PLATFORM=minimal
                    UITK_ICON_THEME=suru
                    ${U8TEST_ENVIRONMENT}
```

---

Now it's a little weird; the one here isn't changed, but the one below is. If you decide to change, please change all instances.